Storage/src/Bucket.php

```diff
@@ -1167,9 +1167,21 @@ public function isWritable($file = null)
      *
      * Please note that this method may not yet be available in your project.
      *
+     * To request a policy with conditions, pass an array with
+     * '[requestedPolicyVersion => 3]' as argument to the policy() and
+     * reload() methods.
+     *
+     * @see https://cloud.google.com/iam/docs/policies#versions policy versioning.
```

Comment:
If we move this `@see` annotation down along with the rest (below line 1187) it should fix the snippet error. It looks like it is having trouble parsing the docblock.

Reply:
I just figured it out 😛after numerous delete and reinserting.